packages/airnode-validator/templates/0.4/config.json

```diff
@@ -133,6 +133,12 @@
             }
```

Comment:
You really don't need to update these and definitely not old ones. You would need to add a new 0.6 one but don't, we'll be removing these soon.

Reply:
There are still some unit tests that failed if I didn't update them :thinking:

Reply:
@Siegrift You've added the new validator, do you know what's going on there?

Reply:
This should only affect old validator, but I'll take a look

Reply:
I see the problem now. The old validator tests are still in place, but we no longer have the 0.6 template and we actually use the new validator in deployer and node. Since you have added a new field to the config.json the tests broke because they are validating it with 0.5 template.

While investigating this I accidentally changed the node version to 0.4 for the 0.5 templates. I do not thing we need to release a fix for v0.5 though (because the new validator does not check the node version).

I think the easiest way to fix this is to copy the `0.5` templates to `0.6` and add the `fulfillmentGasLimit` property there and fix the incorrect versions. See: https://github.com/api3dao/airnode/tree/gas-limit-emo 

I think we should fix the validator properly in v0.7.

Reply:
Can't we comment out those tests that are using the old validator and not update the templates? I agree with finishing this properly in v0.7.

Reply:
Yeah, we can do that as well.

Reply:
@vponline Let's do that then.

Reply:
@Siegrift  This means we need to remove all tests from 
**airnode-validator/src/validator.test.ts** and
**airnode-validator/test/validator.test.ts**, right?

So I'll need to remove the files since empty test files are not allowed.

Reply:
Yeah.